tests/unit/base/test_rest_manager.py

```diff
@@ -0,0 +1,30 @@
+from gitlab import base
+from tests.unit import helpers
```

Comment:
for the most part, this module and `test_rest_object.py` just use functions instead of methods now, and reuse the common fake classes (which are also faked separately in mixin tests.. haven't touched that). So the diff is a bit misleading, it's mostly copy/paste and adding the `helpers.` prefix.